Warning to silence

```
[ 30%] Building C object src/core/CMakeFiles/lxqt-archiver-core.dir/fr-proc-error.c.o
/tmp/makepkg/lxqt-archiver-git/src/lxqt-archiver/src/core/fr-archive.c: In function ‘compute_base_path’:
/tmp/makepkg/lxqt-archiver-git/src/lxqt-archiver/src/core/fr-archive.c:2936:18: warning: assignment discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 2936 |         name_end = strchr (base_path, '/');
      |

--- a/src/core/CMakeLists.txt
+++ b/src/core/CMakeLists.txt
@@ -4,6 +4,9 @@ include_directories(
     ${CMAKE_BINARY_DIR}
 )
 
+# We use GNU extensions
+set(CMAKE_C_EXTENSIONS ON)
+
 add_library(lxqt-archiver-core STATIC
     tr-wrapper.c  # our own wrapper for QTranslater
     file-data.c

